feathr_project/feathrcli/data/feathr_user_workspace/nyc_driver_demo.ipynb

```diff
@@ -236,10 +234,7 @@
         "    port: 6380\n",
         "    ssl_enabled: True\n",
         "feature_registry:\n",
-        "  purview:\n",
-        "    type_system_initialization: true\n",
-        "    purview_name: 'feathrazuretest3-purview1'\n",
-        "    delimiter: '__'\n",
+        "  api_endpoint: \"https://feathr-sql-registry.azurewebsites.net/api/v1\"\n",
```

Comment:
We should direct them to the customer's own registry, rather than our demo registry?

Reply:
You are correct. However, the config before this PR did not use customer's registry too. The main purpose of this PR is adding docs for new config and ensure old config not show up for new users.

I take a look and it seems there are a few other resources name hard coded in notebook as well, I opened a new issue #516 to track.

Reply:
Sorry I didn't see the PR (516 should be another one), but assuming that will happen in the new PR.

Reply:
Sorry, I meant issue #546 actually. You are right, there will be a separated PR to clean up all hard coded resource names in getting started notebooks.